modin/tests/pandas/utils.py

```diff
@@ -662,9 +662,10 @@ def assert_dtypes_equal(df1, df2):
         lambda obj: isinstance(obj, pandas.PeriodDtype),
     )
 
-    for col in dtypes1.keys():
+    # `test_pivot_table_margins` failed due to usage ``pd.NA`` in column name
+    for idx in range(len(dtypes1)):
```

Comment:
`test_pivot_table_margins` failed due to usage ``pd.NA`` in column name

Reply:
Is it an issue on our side?

Reply:
There was a problem, the new implementation is not affected by this problem.